### PR TITLE
Updates from the past year

### PR DESCRIPTION
**Pull request by hobovsky**
So I closed #11 and created a new PR to hopefully un-break things.

Sorry for all inconvenience.

From the original PR:
> My attempt on the first post announcing changes which happened on Codewars.
>
> I was not sure how much details to put here, so I decided to have just a summary, and a couple of follow-up posts published separately with more details on some topics. Please let me know how it works for you.

### CODE REVIEW

File: content/2021/2020-01-21_whats-new/index.md
Comment (Steffan153) [inline content/2021/2020-01-21_whats-new/index.md]:
```suggestion
The year 2020 and the beginning of 2021 was a very busy period for Codewars maintainers. We worked hard on improving the platform, and on fixing existing bugs that were getting in a way of further improvements. We were looking for ways and ideas on how to get closer to Codewars users. At the same time, we managed to introduce a few new languages to train on, upgraded some language versions, and provided some means to support users and newcomers.
```

File: content/2021/2020-01-21_whats-new/index.md
Comment (Steffan153) [inline content/2021/2020-01-21_whats-new/index.md]:
```suggestion
We migrated the Codewars blog to a new platform, and plan to use it for announcements for the Codewars community. We will post here information and updates about new features, improvements, and we will call for your help when we need one. Blog posts will show up on the dashboard of the main site, which will hopefully help to bring the Codewars closer to its users.
```

File: content/2021/2020-01-21_whats-new/index.md
Comment (Steffan153) [inline content/2021/2020-01-21_whats-new/index.md]:
```suggestion
2020 brought a large set of new languages which users can train on, create new kata, and translate existing kata to! Most of them are still in the beta stage, and contributions in form of new kata or translations will be very helpful. Many languages were also updated to new versions. Some of the added languages are Lean, COBOL, Perl, Raku, and Pascal, but these are not the only ones! More details will be provided in a dedicated post.
```

File: content/2021/2020-01-21_whats-new/index.md
Comment (Steffan153) [inline content/2021/2020-01-21_whats-new/index.md]:
```suggestion
Refactoring was probably the biggest topic of all activities related to Codewars conducted in recent time. Thanks to code cleanup, reducing technical debt, and even total rewrite of some elements, it was possible to improve performance in the backend and improve user experience and navigation on the frontend. Getting rid of some messy code also enabled maintainers to provide further improvements in the future, which, until now, were very difficult to add.
```

File: content/2021/2020-01-21_whats-new/index.md
Comment (Steffan153) [inline content/2021/2020-01-21_whats-new/index.md]:
```suggestion
Addressing the needs of new users and questions of many experienced ones, a new [documentation site](https://docs.codewars.com/) has been launched. Documentation is meant to be created by the community, so everyone can write articles, review drafts, suggest improvements, or fix any issues using [codewars/docs](https://github.com/codewars/docs) GitHub repository. The documentation is still a work in progress, but when it gets into an acceptably good shape, it will replace the old Codewars wiki.
```

File: content/2021/2020-01-21_whats-new/index.md
Comment (Steffan153) [inline content/2021/2020-01-21_whats-new/index.md]:
```suggestion
To take some maintenance burden off of the administrators' back, the [Codewars Community Moderation Team](https://docs.codewars.com/community/moderation/#members) has been formed and provided with a variety of tools to manage content, handle problematic users, support users, and perform other moderation tasks. 
```

File: content/2021/2020-01-21_whats-new/index.md
Comment (Steffan153) [inline content/2021/2020-01-21_whats-new/index.md]:
```suggestion
More information on the moderation team will be provided in a dedicated announcement.
```

File: content/2021/2020-01-21_whats-new/index.md
Comment (Steffan153) [inline content/2021/2020-01-21_whats-new/index.md]:
```suggestion
In the upcoming time, we plan to work on further improvements to the platform, extending tools available for moderators, and creating more articles for documentation. We want to bring Codewars and users closer together, so will keep announcing incoming changes and ask for your help when we need one. We would also like to give a big thanks to our community for continued support and participation.
```

**Review by Steffan153 — CHANGES_REQUESTED** [review]

File: content/2021/2020-01-21_whats-new/index.md
Comment (kazk) [inline content/2021/2020-01-21_whats-new/index.md]:
```suggestion
title = "Updates from the past year"
```

"2020 brought" reminds me of COVID.

Can you change the file name as well? And the date in the filename needs to be fixed to use 2021.

File: content/2021/2021-01-21_updates-from-the-past-year/index.md
Comment (kazk) [inline content/2021/2021-01-21_updates-from-the-past-year/index.md]:
```suggestion
tags = ["announcement", "changelog", "runner", "new-language", "documentation", "moderation"]
```

File: content/2021/2020-01-21_whats-new/index.md
Comment (hobovsky) [reply content/2021/2020-01-21_whats-new/index.md]: The date in the file name was a typo.

File: content/2021/2021-01-21_updates-from-the-past-year/index.md
Comment (kazk) [inline content/2021/2021-01-21_updates-from-the-past-year/index.md]:
```suggestion
For the past few years, we had been focused on the internal changes to fix critical bugs and performance issues. We've been aware of the various problems of the platform including the community issues, but it was necessary for us to prioritize those major technical issues. We're happy to announce that Codewars is now in a much better shape! Thank you for your patience.

We started to focus more on the community aspect since the summer of 2020 and have some exciting announcements to make.
In this post, we'll highlight the updates from the past year and explain our plan for the early 2021.
```

Sorry, I didn't mean to replace the whole paragraph. I first thought it sounded weird to say "Codewars maintainers" in our blog.

File: content/2021/2021-01-21_updates-from-the-past-year/index.md
Comment (kazk) [inline content/2021/2021-01-21_updates-from-the-past-year/index.md]:
```suggestion
Some of you might have noticed already, but we have a new blog. It's [open sourced](https://github.com/codewars/blog) on GitHub to collaborate with the community. Feel free to open issues for any feedback.

We'll be posting about anything related to Codewars, including new features, improvements, and will ask for your help when needed. We're also considering guest posts by users to share their experience in the future.
```

Removed the dashboard because most of them will be coming from there.

File: content/2021/2021-01-21_updates-from-the-past-year/index.md
Comment (kazk) [inline content/2021/2021-01-21_updates-from-the-past-year/index.md]:
```suggestion
We've added support for the following languages: Prolog, CFML, COBOL, Lean, Haxe, CommonLisp, Perl, Raku, and Pascal.

Most of them are still in the beta stage, and contributions in the form of new kata or translations will be very helpful. Many of the existing languages were also updated to newer versions. More details will be provided in a dedicated post.
```

File: content/2021/2021-01-21_updates-from-the-past-year/index.md
Comment (kazk) [inline content/2021/2021-01-21_updates-from-the-past-year/index.md]:
```suggestion
## Refactoring and performance improvements

As mentioned above, we've been mostly focused on the technical issues for the past few years. We're relieved that _most_ of the critical issues have been resolved and the performance have improved significantly. In short, it's now at least 10 times faster on average compared to 2018 with less servers and much more traffic. Thank you for your patience and support if you had experienced the worst days and stayed with us.
```

File: content/2021/2021-01-21_updates-from-the-past-year/index.md
Comment (kazk) [inline content/2021/2021-01-21_updates-from-the-past-year/index.md]:
```suggestion
We've been working on the new [documentation site](https://docs.codewars.com/) to replace the old [wiki](https://github.com/codewars/codewars.com/wiki).

The new docs is [open source](https://github.com/codewars/docs) and maintained by the community moderation team. You're encouraged to participate by writing new articles, reviewing drafts, suggesting improvements, or fixing any issues.
```

Move the section about the community team above this.

File: content/2021/2021-01-21_updates-from-the-past-year/index.md
Comment (kazk) [inline content/2021/2021-01-21_updates-from-the-past-year/index.md]:
```suggestion
More information on documentation will be provided in a dedicated announcement. Any contribution is welcomed!
```

File: content/2021/2021-01-21_updates-from-the-past-year/index.md
Comment (kazk) [inline content/2021/2021-01-21_updates-from-the-past-year/index.md]:
```suggestion
The Codewars community needed moderation for a long time, but it was difficult because we only have one developer/admin. We wanted to delegate various tasks to a team of trusted users, but all the technical issues got in the way...

We're excited to introduce [the community moderation team](https://docs.codewars.com/community/moderation/#members)! They have access to a variety of tools to manage content, handle problematic users, support users, and perform other moderation tasks. 
```

File: content/2021/2021-01-21_updates-from-the-past-year/index.md
Comment (kazk) [inline content/2021/2021-01-21_updates-from-the-past-year/index.md]:
```suggestion
## Plans for 2021
```

File: content/2021/2021-01-21_updates-from-the-past-year/index.md
Comment (kazk) [inline content/2021/2021-01-21_updates-from-the-past-year/index.md]:
```suggestion
In 2021, we plan to work on further improvements to the platform, expanding moderation tools, and creating more articles for the documentation. We'd like to bring Codewars community closer together and improve communication with the community, so we'll keep you updated.
```

**Review by kazk — CHANGES_REQUESTED** [review]
Sorry, I changed a lot. What you had written helped me a lot, though.

I like the idea of having separate posts for details. Feel free to let me know if you disagree with the changes or need clarification.

File: content/2021/2021-01-21_updates-from-the-past-year/index.md
Comment (kazk) [reply content/2021/2021-01-21_updates-from-the-past-year/index.md]: I think we need to get better at tagging in general. I'd like to avoid having too many hashtags like social media.

Comment (hobovsky) [reply content/2021/2021-01-21_updates-from-the-past-year/index.md]: I agree, but this post contains overview of many topics, that's why it ended up with many tags. Other posts would have fewer of them, but we need to agree upon some fixed set of them.

Comment (kazk) [reply content/2021/2021-01-21_updates-from-the-past-year/index.md]: I thought `improvements` was unnecessary and too general. Most updates will be an improvement.

**Comment by hobovsky** [comment]
> Sorry, I changed a lot. What you had written helped me a lot, though.
> 
> I like the idea of having separate posts for details. Feel free to let me know if you disagree with the changes or need clarification.

It's you who's going to post it, and it's meant to express your feelings, so you can do with this whatever you want. I do not write this for me, I just want to support you and help you to collect some thoughts. You can change whatever you want, you do not have to ask me for approval, you can commit changes right away without going through suggestions. It's for you to work with.

There's going to be your name under the post after all. It should look as you see it good.

File: content/2021/2021-01-21_updates-from-the-past-year/index.md
Comment (kazk) [inline content/2021/2021-01-21_updates-from-the-past-year/index.md]:
```suggestion
We're excited to introduce [the community moderation team](https://docs.codewars.com/community/moderation/#moderators)! They have access to a variety of tools to manage content, handle problematic users, support users, and perform other moderation tasks. 
```
It was linked to the admin members. The moderation members have id `members-1`.

Comment (hobovsky) [reply content/2021/2021-01-21_updates-from-the-past-year/index.md]: I think there's still some problem with anchors, I am not sure though. Sometimes I am almost certain I navigate to some section, but the url in the address bar ends up being different.

I will see if I manage to confirm this reliably.

**Review by kazk — APPROVED** [review]
Thank you so much @hobovsky. I'm terrible at writing these, so it's extremely helpful.

I think it looks good enough now.

**Pull request merged**
